src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ErrorRendererPass.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\TwigBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\Asset\Packages;
+use Symfony\Component\DependencyInjection\Alias;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\Emoji\EmojiTransliterator;
+use Symfony\Component\ErrorHandler\ErrorRenderer\CliErrorRenderer;
+use Symfony\Component\ExpressionLanguage\Expression;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Component\Workflow\Workflow;
+use Symfony\Component\Yaml\Yaml;
+use function Symfony\Component\DependencyInjection\Loader\Configurator\inline_service;
+
+/**
+ * @author Javier Eguiluz <javier.eguiluz@gmail.com>
+ */
+class ErrorRendererPass implements CompilerPassInterface
+{
+    public function process(ContainerBuilder $container): void
+    {
+        // in the 'test' environment, use the CLI error renderer as the default one
+        if ($container->hasDefinition('test.client')) {
+            $container->getDefinition('twig.error_renderer.html')
+                ->setArgument(1, new Definition(CliErrorRenderer::class));
```

Comment:
This requires updating the `symfony/twig-bridge` requirement in TwigBundle, to ensure it installs a version accepting any ErrorRendererInterface as the fallback renderer. currently, the requirement is `^6.4|^7.0`, and this will break if the 6.4 or 7.1 bridge is used. Change the requirement to `^7.2`

Reply:
Updated. Thanks for explaining it in detail 🙏